cli/cmd/stat.go

```diff
@@ -340,11 +340,11 @@ func printSingleStatTable(stats map[string]*row, resourceType string, w *tabwrit
 		"LATENCY_P95",
 		"LATENCY_P99",
 		"TLS",
+		"TCP CONN",
```

Comment:
nit: mind naming this `TCP_CONN`, for consistency with the `LATENCY_*` headers. i think this also makes it visually explicit that it's one column.

taking this a step further, `READ BYTES/SEC` could use updating too. if we want to match `kubectl`, it uses dashes instead of underscores.

```bash
$ kubectl get svc
NAME         TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)    AGE
```

Reply:
ah, good spot! I'll change these to dashes!

Reply:
should this be `TCP-CONN` then? and update the latency headers too?

Reply:
sure dashes all around sounds good!

Reply:
(decided I liked the look/readability of the `_` over the `-` so decided to go with that instead)